src/components/Dashboard/RewardsModule.js

```diff
@@ -88,10 +98,15 @@ const RewardsModule = React.memo(function RewardsModule({
 
       if (!rewards) return
 
+      const nonSettledArbitrables = filterSettledRounds(
+        feeRewards.arbitrableFees
+      )
+      const nonSettledAppeals = filterSettledRounds(feeRewards.appealFees)
+
       onClaimRewards(
         wallet.account,
-        feeRewards.arbitrableFees,
-        feeRewards.appealFees,
+        nonSettledArbitrables,
+        nonSettledAppeals,
```

Comment:
Just a bit worried from the comment before in the utils; it looks like the data types for these variables may have changed?

Reply:
Ah no actually the conversion is needed since we changed the data structure for the rounds in `useJurorRewards`.

Before rounds were just an array of `roundId` and now they are an object of `{roundId, settled}` so we needed to convert it back to just an array of `roundId`s. 

Or maybe we could just change how they are handled in `useCourtContracts` so we don't need to do this re conversion ?

Reply:
Ended up removing the already settled roundIds from the fees data structure as we don't really need them. That way we don't need to do the re conversion either.